15-synthesis.Rmd

```diff
@@ -137,7 +137,7 @@ Other topics which received limited attention were remote sensing and using R al
 There are many resources on these topics, including @wegmann_remote_2016 and the GIS-related teaching materials available from [Marburg University](https://moc.online.uni-marburg.de/doku.php).
 
 Instead of covering spatial statistical modeling and inference techniques, we focussed on machine learning\index{machine learning} (see Chapters \@ref(spatial-cv) and \@ref(eco)).
-Again, the reason was that there are already excellent resources on these topics, especially with ecological use cases, including @zuur_mixed_2009, @zuur_beginners_2017 and freely available teaching material and code on *Geostatistics & Open-source Statistical Computing* by David Rossiter, hosted at [css.cornell.edu/faculty/dgr2](http://www.css.cornell.edu/faculty/dgr2/teach/degeostats.html).
+Again, the reason was that there are already excellent resources on these topics, especially with ecological use cases, including @zuur_mixed_2009, @zuur_beginners_2017 and freely available teaching material and code on *Geostatistics & Open-source Statistical Computing* by David Rossiter, hosted at [css.cornell.edu/faculty/dgr2](http://www.css.cornell.edu/faculty/dgr2/teach/degeostats.html) and the [*granolarr*](https://sdesabbata.github.io/granolarr/) project by [Stefano De Sabbata](https://stefanodesabbata.com/) [at the University of Leicester](https://www2.le.ac.uk/departments/geography/people/stefano-de-sabbata) for an introduction to R\index{R} for geographic data science\index{data science}.
```

Comment:
Couldn't have said it better myself. Fits perfectly, thanks for taking the time to slot it into the ideal place :+1: